pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -352,3 +399,7 @@ func renderCrioConfigSnippet(profile *performancev2.PerformanceProfile, src stri
 
 	return crioConfig.Bytes(), nil
 }
+
+func stringSplit(cpuListSlice []int) string {
+	return strings.Trim(strings.Join(strings.Fields(fmt.Sprint(cpuListSlice)), ","), "[]")
+}
```

Comment:
```
func listToString(cpus []int) string {
    items := make([]string, len(cpus))
    for idx, cpu := range cpus {
        items[idx] = strconv.FormatInt(cpu, 10)
    }
    return strings.Join(cpus, ",")
}
```
(this will require a unit test)